### DISCUSSION TITLE
Fresh install with hacmini 3.2.2 causes reboots and/or freezes

### DISCUSSION THREAD

**Discussion by bypo** [osy86/HaC-Mini#1046] [Q&A]
Hi,

My NUC was working correctly with MacOS Ventura. I installed it from scratch with Monterey and then upgraded it, along with each hacmini version.
Today i decided to do a clean installation creating a fresh new USB Key with Ventura 13.0.1+hacmini 3.2.2 and this happened:
- The installation succeded without any issue
- I installed hacmini 3.2.2 and then, when logging in, the system will not last more than 1-2 minutes. Then it will freeze or reboot.
This is something related to hacmini 3.2.2

I tried reinstalling MacOS Monterey + hacmini 3.1.5 and everything is working correctly. Is it a bug or am i doing something wrong?

**Comment by osy** [comment]
Did you get a crash log after it restarts?

**Reply by bypo** [reply]
Nothing useful in the logs.
I noticed that if i boot Recovery partition, the system is stable for hours (even if i can just open disk utility and Safari).

Any hints on how to debug?

**Reply by osy** [reply]
When your system reboots do you see a popup asking if you want to submit a crash report to apple? Click more details and it should show you the crash log.

**Reply by bypo** [reply]
Nope, it reboots without any notice :(
I only get a screen at reboot telling me that the system has crashed...

But why during the installation and in recovery mode the system is stable and once installed it reboots?
Any guess? How could i isolate the origin of this crash?

**Reply by osy** [reply]
I'm sorry for pushing this but I think you may have missed it. Sometimes the popup appears behind all the windows that gets restored after your computer restarts. If in doubt, close every window after it restarts and see if there's a dialog behind.

**Comment by bypo** [comment]
You don't have to be sorry. You were right. Here you find a couple of pictures i have been able to take after the restart, then it rebooted.
![1670357638581](https://user-images.githubusercontent.com/14150001/206013803-2a7c90d2-b3a5-4c4e-9c85-b29b52dbe0f5.jpg)
![1670357638585](https://user-images.githubusercontent.com/14150001/206013819-0b467261-1115-4efe-aeb7-8a2c4672a68c.jpg)

Here you find the pictures

**Reply by bypo** [reply]
Ops sorry, this happened after installing Hacmini 3.2.2 on a fully functional MacOS X 12.6.1 that was running smoothly on Hacmini 3.1.5.

Hope it helps!

**Comment by bypo** [comment]
Here you'll find a couple of additional screenshot taken during one of the crashes.

![1670358407935](https://user-images.githubusercontent.com/14150001/206015775-81a0ac53-3eac-4fe6-8e10-c675caf375e9.jpg)


![1670358236785](https://user-images.githubusercontent.com/14150001/206015427-21a05d47-805f-4eb2-bad9-c3be3909d9b1.jpg)

**Reply by osy** [reply]
I’m 80% sure your SSD is overheating. I’ve had similar issues. Make sure your thermal pad is touching the SSD and consider replacing it. Also there’s known issues with many NVMe models and Hackintosh. Check out https://dortania.github.io/Anti-Hackintosh-Buyers-Guide/Storage.html

**Reply by bypo** [reply]
OSY,

It's not something related to my HW. Right now i'm using my Hackmini running MacOS X 12.6.1 + hackmini 3.1.5 without any issues at all. I also tried to install hackmini 3.2.1 but he didn't reboot and with 3.2.2 is still crashing.
I'm running this setup for some weeks and then i'll upgrade to Ventura to see what's happening.

Previously i was running Ventura (upgraded from Monterey without any issues).
To me this seems something related to some settings inherited from previous versions that is not happening on fresh installs on Ventura but i might be wrong.

Thank you for all your work. This project is amazing!
Oreste

**Reply by osy** [reply]
Maybe you chose different options like using NVMe fix for one install and not for another.

**Comment by bypo** [comment]
I'm planning to check the settings.

Keep you posted by tomorrow

Il giorno mar 6 dic 2022 alle ore 22:36 osy ***@***.***> ha
scritto:

**Reply by bypo** [reply]
Checked it now. Here are what i used

Installed MacOS X 12.6.1 
HaCmini 3.1.5 - Default (No Crashes - Everything OK!)
HaCmini 3.2.1 - Default settings (Doesn't boot OS - bootloop)
HaCmini 3.2.2 - Default settings (Boots OS but then it crashes after 2-3 minutes)

Right now MacOS X 12.6.1 + 3.1.5 is stable and fully working on the same HW.

Cheers
